lib/libutils/isoc/include/malloc.h

```diff
@@ -89,4 +89,7 @@ size_t malloc_get_max_allocated(void);
 size_t malloc_get_allocated(void);
 size_t malloc_get_heap_size(void);
 
+void __malloc_lock(void);
+void __malloc_unlock(void);
```

Comment:
I can't declare `extern struct mutex __mallloc_mu` in `malloc.h` as that would require `<mutex.h>` to be included, which isn't available for user TAs. I could do an extern declaration in `bget_malloc.c` instead of adding a `malloc_private.h` for this sole purpose.

Please advice.